javatests/build/bazel/tests/integration/BazelBaseTestCaseTest.java

```diff
@@ -54,21 +52,16 @@ public void testRunningBazelInGivenRelativeDirectory() throws Exception {
     driver.scratchFile("foo/BUILD", "sh_test(name = \"bar\",\n" + "srcs = [\"bar.sh\"])");
     driver.scratchExecutableFile("foo/bar.sh", "echo \"in bar\"");
 
-    Command cmd = driver.bazelCommand("run", "bar").inWorkingDirectory(Paths.get("foo")).build();
-
-    assertEquals(0, cmd.run());
-    assertThat(cmd.getOutputLines()).contains("in bar");
+    BazelCommand cmd = driver.bazel("run", "bar").inWorkingDirectory(Paths.get("foo")).mustRun();
+    assertThat(cmd.outputLines()).contains("in bar");
   }
 
   @Test
   public void testTestSuiteExists() throws Exception {
     loadIntegrationTestRuleIntoWorkspace();
     setupPassingTest("IntegrationTestSuiteTest");
 
-    Command cmd = driver.bazelCommand("test", "//:IntegrationTestSuiteTest").build();
-    final int exitCode = cmd.run();
-
-    org.hamcrest.MatcherAssert.assertThat(exitCode, is(successfulExitCode(cmd)));
```

Comment:
should we delete `successfulExitCode`?

Reply:
it's important to note we're losing here:
workspaceDirectoryContents and logs of tests.
They were very valuable for me while developing but maybe we can live with them.
As a thought exercise (and maybe as input for us as maintainers of the lib) how should one write this test if they also want the above as diagnostics when it fails?

Reply:
I delete successfulExitCode.

Logs of tests is something I wanted to be accessible through the Build Event Protocol interface.  In BazelCommand you have a toString() method which is used to describe the command in mustRun(), and it can be used in general for debugging.  Right now, the toString() method only gives the stderr and stdout, but I planned with the Build Event Protocol support to expand it to giving a list of produced artifacts and test logs.

I don't remember having in the workspaceDirectoryContents as well, so I just added it to toString() as well.